codes/data/usa/merge_policy_and_cases.py

```diff
@@ -101,16 +101,17 @@ def min_no_nans(x):
 
 def download_and_process_policy_csv():
 
-	policy_data_raw = pd.read_csv(os.path.join(raw_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
+	policy_data_raw = pd.read_csv(os.path.join(int_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
```

Comment:
Note that this i/o will need to change to read from both `data/raw/USA_policy_data_sources.csv` and `data/interim/USA_policy_data_sources_other` and append these (as in the Policy Data section in #21 ). @ekrasovich and @jeanettelt can help advise on this.

Reply:
yeah i can do this on a future PR!